chore: polish kb52-62 files (#1006)
keymanapp/help.keyman.com: knowledge-base/kb0059.md

# HOWTO: Configure SIL Toolbox, Keyman, and Windows keyboards optimally

### **NOTE**: This archived documentation has not been updated recently and may contain information that is no longer relevant

SIL Toolbox includes automatic keyboard switching for its languages.  However, due to the way keyboards are selected in Toolbox, you cannot use Keyman Configuration's Languages tab to link Keyman keyboards to the Windows languages you are using.  The following instructions should help.

In these instructions, we are using the SIL IPA Unicode keyboard and linking it to the Icelandic language.  Substitute these names accordingly when you are configuring your own keyboards.

- (Desktop Pro only) Remove the Keyman Desktop Text Services Framework Add-in.  Toolbox does not work with the Text Services Framework correctly.  To remove the Add-in:

>    - Start **Control Panel**, **Add/Remove Programs** (this is now called **Programs and Features** in Vista)
>    - Find **Tavultesoft Keyman Desktop** and click **Change**.
>    - Click **Next**, then **Change** in the install wizard that appears.
>    - Expand the **Addins** node and change the setting for **Text Services Framework Addin** to **Uninstall** or **Not Installed**
>    - Click **Next** to apply the changes
>    - Note: until you restart Windows, the Addin may still be visible in some aspects 
>        of the Windows user interface.

Author: ```suggestion
   i. Start **Control Panel**, **Add/Remove Programs** (this is now called **Programs and Features** in Vista)
   ii. Find **Tavultesoft Keyman Desktop** and click **Change**.
   iii. Click **Next**, then **Change** in the install wizard that appears.
   iv. Expand the **Addins** node and change the setting for **Text Services Framework Addin** to **Uninstall** or **Not Installed**
   v. Click **Next** to apply the changes
   vi. Note: until you restart Windows, the Addin may still be visible in some aspects of the Windows user interface.

```

- Add the SIL IPA Unicode Keyman keyboard as a System Shadow Keyboard (the SIL IPA Unicode Keyman keyboard must be installed for all users - if it isn't you must uninstall the SIL IPA Unicode Keyman keyboard and reinstall it for all users - this is a Windows requirement in order for the SIL IPA Unicode Keyman keyboard to be visible in the list of Windows keyboards):

>     - Right-click on the **Language Bar** and select **Settings**.
>     - Click Add, select **Icelandic**, expand the **Keyboards** node, and select **Show More**.
>     - Select **SIL IPA Unicode** from the expanded list of keyboards and click **OK**.
>     - Click **OK** again to save changes.  

Author: ```suggestion
    i. Right-click on the **Language Bar** and select **Settings**.
    ii. Click Add, select **Icelandic**, expand the **Keyboards** node, and select **Show More**.
    iii. Select **SIL IPA Unicode** from the expanded list of keyboards and click **OK**.
    iv. Click **OK** again to save changes.  

```

- In Toolbox, configure your language settings:

>    - Click **Project**, **Language Encodings**
>    - Select the language you wish to link, and click **Modify**.
>    - Click **Options**, and in the **Keyboard** combo box, select **SIL IPA Unicode**
>    - Click **OK**, **Close** to save changes.

Author: ```suggestion
   i. Click **Project**, **Language Encodings**
   ii. Select the language you wish to link, and click **Modify**.
   iii. Click **Options**, and in the **Keyboard** combo box, select **SIL IPA Unicode**
   iv. Click **OK**, **Close** to save changes.

```

# HOWTO: Configure SIL Toolbox, Keyman, and Windows keyboards optimally

### **NOTE**: This archived documentation has not been updated recently and may contain information that is no longer relevant

SIL Toolbox includes automatic keyboard switching for its languages.  However, due to the way keyboards are selected in Toolbox, you cannot use Keyman Configuration's Languages tab to link Keyman keyboards to the Windows languages you are using.  The following instructions should help.

In these instructions, we are using the SIL IPA Unicode keyboard and linking it to the Icelandic language.  Substitute these names accordingly when you are configuring your own keyboards.

> - (Desktop Pro only) Remove the Keyman Desktop Text Services Framework Add-in.  Toolbox does not work with the Text Services Framework correctly.  To remove the Add-in:
>    i. Start **Control Panel**, **Add/Remove Programs** (this is now called **Programs and Features** in Vista)
>    ii. Find **Tavultesoft Keyman Desktop** and click **Change**.
>    iii. Click **Next**, then **Change** in the install wizard that appears.
>    iv. Expand the **Addins** node and change the setting for **Text Services Framework Addin** to **Uninstall** or **Not Installed**
>    v. Click **Next** to apply the changes
>    vi. Note: until you restart Windows, the Addin may still be visible in some aspects of the Windows user interface.

Reviewer: Might need to use numbers to get the sublist to format

```suggestion
- (Desktop Pro only) Remove the Keyman Desktop Text Services Framework Add-in.  Toolbox does not work with the Text Services Framework correctly.  To remove the Add-in:
  1. Start **Control Panel**, **Add/Remove Programs** (this is now called **Programs and Features** in Vista)
  2. Find **Tavultesoft Keyman Desktop** and click **Change**.
  3. Click **Next**, then **Change** in the install wizard that appears.
  4. Expand the **Addins** node and change the setting for **Text Services Framework Addin** to **Uninstall** or **Not Installed**
  5. Click **Next** to apply the changes
  6. Note: until you restart Windows, the Addin may still be visible in some aspects of the Windows user interface.
```

- Add the SIL IPA Unicode Keyman keyboard as a System Shadow Keyboard (the SIL IPA Unicode Keyman keyboard must be installed for all users - if it isn't you must uninstall the SIL IPA Unicode Keyman keyboard and reinstall it for all users - this is a Windows requirement in order for the SIL IPA Unicode Keyman keyboard to be visible in the list of Windows keyboards):

>     i. Right-click on the **Language Bar** and select **Settings**.
>     ii. Click Add, select **Icelandic**, expand the **Keyboards** node, and select **Show More**.
>     iii. Select **SIL IPA Unicode** from the expanded list of keyboards and click **OK**.
>     iv. Click **OK** again to save changes.  
> 
> 
> - In Toolbox, configure your language settings:
>    i. Click **Project**, **Language Encodings**
>    ii. Select the language you wish to link, and click **Modify**.
>    iii. Click **Options**, and in the **Keyboard** combo box, select **SIL IPA Unicode**
>    iv. Click **OK**, **Close** to save changes.

Reviewer: and here

```suggestion
  1. Right-click on the **Language Bar** and select **Settings**.
  2. Click Add, select **Icelandic**, expand the **Keyboards** node, and select **Show More**.
  3. Select **SIL IPA Unicode** from the expanded list of keyboards and click **OK**.
  4. Click **OK** again to save changes.  

- In Toolbox, configure your language settings:
   1. Click **Project**, **Language Encodings**
   2. Select the language you wish to link, and click **Modify**.
   3. Click **Options**, and in the **Keyboard** combo box, select **SIL IPA Unicode**
   4. Click **OK**, **Close** to save changes
```
